doc/distrib/NodeHelpFiles/_review/DSRevitNodesUI.ViewDuplicateOptions.md

```diff
@@ -0,0 +1,13 @@
+## In Depth
+`View Duplicate Options` provides access to all the duplication options available in Revit. 
+
+These include:
+- As Dependent
+- With Detailing
+- As Duplicate
+
+In the example below, the detail view "North Wall Window Detail" is duplicated with a prefix of "-Revision 1" with detailing.
```

Comment:
The node in the example is View.DuplicateView. But the description and file name is for View Duplicate Options. Looks like its been mixed up?